pages/docs/reference/lambdas.md

```diff
@@ -430,6 +430,21 @@ print(sum)
 
 </div>
 
+For lambdas that don't capture any variables (also know as _stateless_ lambdas), there is a built-in optimization:
+the compiler creates only one instance for each such lambda. So, all instances of a stateless lambda point to the same instance.
```

Comment:
> all instances of a stateless lambda point to the same instance.

This confuses the matter: on one hand there is the single instance, on the other—there are many ('all') of them.